src/mimefactory.rs

```diff
@@ -491,6 +491,11 @@ impl<'a> MimeFactory<'a> {
                 "Auto-Submitted".to_string(),
                 "auto-replied".to_string(),
             ));
+        } else if context.get_config_bool(Config::Bot).await? {
+            headers.unprotected.push(Header::new(
+                "Auto-Submitted".to_string(),
+                "auto-generated".to_string(),
+            ));
```

Comment:
I guess headers.unprotected is for headers that are not in the pgp encrypted part? Do we want to put it there if possible? I guess there are other headers which would be more interesting but still want to mention it.